### PR TITLE
Make it possible to feed a list of entities

### PR DESCRIPTION
**Pull request by dgomes**
This PR makes it possible to feed in a list of entities

Like many people I have sensors for the grid and panels, but not for the house (which is the sum of grid and panels)

This enables you to define an entity as being the sum of several entities (no need to create a template sensor that pollutes HA), and is 100% back-compatible (still can supply just a single entity using the current syntax)

Eg.

```
      - type: "custom:tesla-style-solar-power-card"
        grid_consumption_entity: sensor.power_from_the_network
        solar_yield_entity: sensor.power_produced
        house_consumption_entity:
          - sensor.power_from_the_network
          - sensor.power_produced
        grid_feed_in_entity: sensor.power_to_the_network
        car_charging_entity: sensor.garage
```

### CODE REVIEW

File: tesla-style-solar-power-card.js [hunk 1]
```diff
@@ -678,4 +683,4 @@ class TeslaStyleSolarPowerCard extends HTMLElement {
   }
 }
 
-customElements.define('tesla-style-solar-power-card', TeslaStyleSolarPowerCard);
\ No newline at end of file
+customElements.define('tesla-style-solar-power-card', TeslaStyleSolarPowerCard);
```
Comment (dgomes) [inline tesla-style-solar-power-card.js:686]:
```suggestion
customElements.define('tesla-style-solar-power-card', TeslaStyleSolarPowerCard);```

**Comment by reptilex** [comment]
The problem with this is that you cannot create a visual config editor anymore and I'm thinking whether it's not better to create template sensors and sum them up there instead of with the config. Don't you think?

**Comment by dgomes** [comment]
Template Sensors pollute your HA state machine, wasting resources and delaying triggers... 

These template sensors will only be created for the purpose of this wonderful UI :) so I think it fits better to sum it all up in the UI.

How were you think about creating a visual config editor ?

**Comment by reptilex** [comment]
Well in that case I would agree that it is a good idea, but how would you deal with the visual configuration editor in that case?

**Comment by dgomes** [comment]
You keep talking about a visual config editor, but I still don't understand how you plan to build one, can you explain your plan ?

**Comment by reptilex** [comment]
Home assistant has yaml and visual configuration editors. You see visual editors when you use the entity or the gauge card, but mine has only a yaml editor. The new one uses the home assistant editor. I figured a way now, so I will add this soon.

**Comment by dgomes** [comment]
I think that level requires the card to be part of the Official Distribution and not a custom component, but I'm not a frontend contributor to say that for sure.

But please keep yaml for those of us who prefer to use it instead of the GUI ;)

**Pull request merged**